src/studio/src/designer/Dockerfile

```diff
@@ -30,26 +34,32 @@ RUN yarn workspace dashboard run build
 # Create altinn-studio-frontend base image
 FROM alpine as altinn-studio-frontend
 WORKDIR /dist/
-COPY --from=generate-studio-frontend /dist/app-development/app-development.js .
-COPY --from=generate-studio-frontend /dist/app-development/1.app-development.js .
-COPY --from=generate-studio-frontend /dist/app-development/2.app-development.js .
-COPY --from=generate-studio-frontend /dist/app-development/3.app-development.js .
-COPY --from=generate-studio-frontend /dist/app-development/4.app-development.js .
-COPY --from=generate-studio-frontend /dist/app-development/editor.worker.js .
-COPY --from=generate-studio-frontend /dist/app-development/ts.worker.js .
-COPY --from=generate-studio-frontend /dist/app-development/app-development.css .
-COPY --from=generate-studio-frontend /dist/dashboard/dashboard.js .
-COPY --from=generate-studio-frontend /dist/dashboard/dashboard.css .
+COPY --from=generate-studio-frontend /build/dist/app-development/app-development.js .
+COPY --from=generate-studio-frontend /build/dist/app-development/1.app-development.js .
+COPY --from=generate-studio-frontend /build/dist/app-development/2.app-development.js .
+COPY --from=generate-studio-frontend /build/dist/app-development/3.app-development.js .
+COPY --from=generate-studio-frontend /build/dist/app-development/4.app-development.js .
+COPY --from=generate-studio-frontend /build/dist/app-development/editor.worker.js .
+COPY --from=generate-studio-frontend /build/dist/app-development/ts.worker.js .
+COPY --from=generate-studio-frontend /build/dist/app-development/app-development.css .
+COPY --from=generate-studio-frontend /build/dist/dashboard/dashboard.js .
+COPY --from=generate-studio-frontend /build/dist/dashboard/dashboard.css .
 
 FROM node:16.13.1-alpine3.14 AS generate-designer-js
 WORKDIR /app
 COPY src/designer/backend/package.json .
 COPY src/designer/backend/yarn.lock .
-COPY src/designer/backend .
+COPY src/designer/backend/.yarn ./.yarn
+COPY src/designer/backend/.yarnrc.yml .
+
+RUN corepack enable
```

Comment:
Probably don't have to do this twice? Details..

Reply:
This is a separate stage in a multi-stage-build, which doesnt know about anything before line 48. This needs to be executed here as well as on line 18.